Formula/ruby@2.3.rb

```diff
@@ -117,7 +99,7 @@ def program_suffix
   end
 
   def rubygems_bindir
-    "#{HOMEBREW_PREFIX}/bin"
+    "#{HOMEBREW_PREFIX}/lib/ruby/gems/#{abi_version}/bin"
```

Comment:
what's going on here?

Reply:
It's what we do for all the versioned Rubies, beside this one. I think that was a likely screw up whenever this was added as a versioned formulae.

Reply:
:( ok

Reply:
Just prevents the main `ruby` from having its gems accidentally smashed by the older `ruby@whatever` I guess.